cypress/integration/teacher-training-adviser.spec.js

```diff
@@ -134,20 +131,20 @@ describe("Feature - Get an adviser : Tests execution date and time : " + new Dat
 		  Do you have grade 4 (C) or above in GCSE science, or equivalent? - Yes
 		  UK user
 		*/
-		var stage;
+
 		cy.enterFirstNameLastNameandEmail(
 			this.testData.firstName,
 			this.testData.lastName,
 			this.testData.email
 		);
-		cy.returningToTeaching((returner = false));
+		cy.returningToTeaching(false);
 		cy.doYouHaveDegree("Yes");
 		cy.selectWhatSubjectIsYourDegree("Dance");
 		cy.selectWhichClassIsYourDegree("2:2");
 		cy.wait(1000);
 		cy.selectStage("Primary");
 		cy.gcseMathsAndEnglish(true);
-		cy.doYouHaveGrade4CorAboveInGCSEScienceorEquivalent("Yes");
+		cy.doYouHaveGrade4CorAboveInGCSEScienceorEquivalent(true);
```

Comment:
I'd be tempted to simplify this to match the other method, so `cy.gcseScience(true)`

Reply:
Resolved